### DISCUSSION TITLE
Unintentional translation of entity names when reinstalling Huawei Solar

### DISCUSSION THREAD

**Discussion by rapahl** [wlcrs/huawei_solar#703] [General]
If I want to reinstall the Huawei solar integration (new IP) all entities are translated into German. This means I lose my history. Is there a way to keep the entities in English. I didn't find any results using the search function. Thank you for helpful answers.
Rainer

**Comment by jires** [comment]
You can rename entities id's manually to the old one names, if they are not existing in the system, so history will come back. Just keep in mind,  when you remove devices and will add again, it will revert back to default names

**Reply by rapahl** [reply]
Thanks for the answer, I'll try that. Of course it would be nicer if I could activate the English translation.
